ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -169,3 +169,209 @@ def verification_ui(self):
+                        locator=self.dep_loc["warning-alert"],
+                        expected_text="Refresh web console",
+                    )
+                    if refresh_web_console_popup:
+                        logger.info(
+                            "Refresh web console option is now available, click on it to see the changes"
+                        )
+                        self.do_click(self.dep_loc["refresh-web-console"])
+            self.navigate_odf_overview_page()
+            logger.info(
+                "Navigate to System Capacity Card and Click on 'ocs-storagecluster-storagesystem'"
+            )
+            self.do_click(self.dash_items["odf-capacityCardLink"])
+            navigate_to_storagesystem_details_page = self.check_element_text(
+                "StorageSystem details"
+            )
+            if navigate_to_storagesystem_details_page:
+                logger.info(
+                    "Successfully navigated to 'StorageSystem details' page from System Capacity Card"
+                )
+            else:
+                logger.error(
+                    "Couldn't navigate to 'StorageSystem details' page from System Capacity Card"
+                )
+            logger.info("Click on StorageSystems breadcrumb")
+            self.do_click((self.dash_items["storagesystems"]))
+            logger.info("Navigate back to ODF Overview page")
+            self.do_click((self.dash_items["overview"]))
+            logger.info(
+                "Now navigate to Performance Card and Click on 'ocs-storagecluster-storagesystem'"
+            )
+            self.do_click(self.dash_items["odf-performanceCardLink"])
+            navigate_to_storagesystem_details_page = self.check_element_text(
+                "StorageSystem details"
+            )
+            if navigate_to_storagesystem_details_page:
+                logger.info(
+                    "Successfully navigated to 'StorageSystem details' page from Performance Card"
+                )
+            else:
+                logger.error(
+                    "Couldn't navigate to 'StorageSystem details' page from Performance Card"
+                )
+            logger.info("Now again click on StorageSystems breadcrumb")
+            self.do_click((self.dash_items["storagesystems"]))
+            logger.info("Navigate again to ODF Overview page")
+            self.do_click((self.dash_items["overview"]))
+            self.page_has_loaded(retries=15, sleep_time=5)
+            logger.info(
+                "Successfully navigated back to ODF tab under Storage, test successful!"
+            )
+
+    def odf_storagesystems_ui(self):
+        """
+        Test to verify changes and validate elements on ODF Storage Systems tab for ODF 4.9
+
+        """
+        self.ocp_version = get_ocp_version()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if self.ocp_version >= "4.9" and ocs_version >= version.VERSION_4_9:
```

Comment:
Enabling console plugin is added in the function odf_overview_ui. Make this a separate function. This is required for almost all UI tests except install and upgrade.